mobile/status.go

```diff
@@ -57,6 +59,15 @@ import (
 	"github.com/status-im/status-go/signal"
 )
 
+var (
+	lastGCTime     time.Time
+	lastGCTimeLock sync.Mutex
```

Comment:
We should avoid creating anything globally. Perhaps `GethStatusBackend` should be the right place for this now.

Also, the test `TestSwitchToLowMemoryMode` you introduced modifies global variables and relies on it. It won't work with in nightly tests with `-test.count`, as it runs the tests in parallel.

[Code Smell #8: Modify global variables in tests](https://www.notion.so/Code-Smell-8-Modify-global-variables-in-tests-2996491814fb495b8ef0ceeb4e657798?pvs=4)